src/Kernel32/storebanned/Kernel32+SafeThreadHandle.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright © .NET Foundation and Contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#pragma warning disable SA1625 // Element documentation must not be copied and pasted
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+    using Microsoft.Win32.SafeHandles;
+
+    /// <summary>
+    /// Contains definition for <see cref="SafeThreadHandle"/>
+    /// </summary>
+    public partial class Kernel32
+    {
+        /// <summary>
+        /// A <see cref="SafeHandle"/> for thread handles.
+        /// </summary>
+        public class SafeThreadHandle : SafeHandleZeroOrMinusOneIsInvalid
```

Comment:
We already have `SafeObjectHandle` which is implemented in very nearly the same way, except that `-1` isn't considered invalid for `SafeObjectHandle`. Can we reuse that instead of defining a new SafeHandle type?

Reply:
That’s the 2nd (unstated) reason why I flipped the PR from active to draft :-). I realized this myself as I was working through the tests.